aries-site/src/layouts/content/ExampleControls.js

```diff
@@ -8,6 +8,7 @@ const ControlButton = ({ children, ...rest }) => (
     {({ hover }) => (
       <Box
         direction="row"
+        align="center"
```

Comment:
On mobile, I think I'd expect the "see full screen" button to be at the top of the right. What do you think?
<img width="624" alt="Screen Shot 2020-04-01 at 3 45 34 PM" src="https://user-images.githubusercontent.com/12522275/78275900-181ba400-74c7-11ea-9e73-e7df80e8bc0b.png">